scripts/pylint_extensions_test.py

```diff
@@ -163,8 +472,106 @@ def test(test_var_one, test_var_two): #@
+                node=missing_param_func_node,
+                args=('args, kwargs',),
+            ),
+        ):
+            self.checker_test_object.checker.visit_functiondef(
+                missing_param_func_node)
+
+        missing_param_func_node = astroid.extract_node("""
+        def func(test_var_one, test_var_two): #@
+            \"\"\"Function to test docstring parameters.
+
+            Args:
+                test_var_one: int. First test variable.
+                invalid_var_name: str. Second test variable.
+
+            Returns:
+                int. The test result.
+            \"\"\"
+            result = test_var_one + test_var_two
+            return result
+        """)
+        with self.checker_test_object.assertAddsMessages(
+            testutils.Message(
+                msg_id='missing-param-doc',
+                node=missing_param_func_node,
+                args=('test_var_two',),
+            ), testutils.Message(
+                msg_id='missing-type-doc',
+                node=missing_param_func_node,
+                args=('test_var_two',),
+            ), testutils.Message(
+                msg_id='differing-param-doc',
+                node=missing_param_func_node,
+                args=('invalid_var_name',),
+            ), testutils.Message(
+                msg_id='differing-type-doc',
+                node=missing_param_func_node,
+                args=('invalid_var_name',),
+            ),
+        ):
+            self.checker_test_object.checker.visit_functiondef(
+                missing_param_func_node)
+
+        class_node, multiple_constructor_func_node = astroid.extract_node("""
+        class Test(): #@
+            \"\"\"Function to test docstring parameters.
+
+            Args:
+                test_var_one: int. First test variable.
+                test_var_two: str. Second test variable.
+
+            Returns:
+                int. The test result.
+            \"\"\"
+
+            def __init__(self, test_var_one, test_var_two): #@
+                \"\"\"Function to test docstring parameters.
+
+                Args:
+                    test_var_one: int. First test variable.
+                    test_var_two: str. Second test variable.
+
+                Returns:
+                    int. The test result.
+                \"\"\"
+                result = test_var_one + test_var_two
+                return result
+        """)
+        with self.checker_test_object.assertAddsMessages(
+            testutils.Message(
+                msg_id='multiple-constructor-doc',
+                node=class_node,
+                args=(class_node.name,),
+            ),
+        ):
+            self.checker_test_object.checker.visit_functiondef(
+                multiple_constructor_func_node)
+
+    def test_visit_raise_warns_unknown_style(self):
+        self.checker_test_object.checker.config.accept_no_raise_doc = False
+        node = astroid.extract_node(
+            """
```

Comment:
Can you please move `"""` above and re-indent this block accordingly?

Reply:
Done